examples/asitvd.js

```diff
@@ -1,15 +1,18 @@
 goog.provide('app.asitvd');
 
-/** @suppress {extraRequire} */
-goog.require('ngeo.mapDirective');
 goog.require('ngeo.source.AsitVD');
 goog.require('ol.Map');
 goog.require('ol.View');
 goog.require('ol.layer.Tile');
 
+goog.require('ngeo.map.module');
 
-/** @type {!angular.Module} **/
-app.module = angular.module('app', ['ngeo']);
+
+/** @type {!angular.Module} */
+app.module = angular.module('app', [
```

Comment:
Create a Jira card so that we do not forget to transform:
-app.module => const appModule

before publishing as es6 modules.

Reply:
Done: https://jira.camptocamp.com/browse/GSGMF-179